api/WebSocket.json

```diff
@@ -23,11 +23,6 @@
               "version_removed": "11",
               "prefix": "Moz",
               "notes": "Message size limited to 16 MB (see <a href='https://bugzil.la/711205'>bug 711205</a>)."
-            },
-            {
-              "version_added": "4",
```

Comment:
Why are you deleting this bit? Is it wrong? Maybe it was prefixed and needs fixing?

Reply:
I tried using the WebSocket API in these earlier versions, but `WebSocket` was undefined in these earlier versions, either prefixed or not.  As such, I've removed this statement.

Reply:
I see, maybe it was behind a flag but not correctly recorded as such.

Reply:
That's quite possible, yeah -- if that's the case though, then it's flag data so old we don't need to keep it around!  :P